test/tests.py

```diff
@@ -6,8 +6,6 @@
 import os
 import logging
 import time
-import requests
-from requests.exceptions import ConnectionError, SSLError
```

Comment:
Actually, the linter correctly pointed out that we still use these: ConnectionError on line 29 and  SSLError on line 35